src/components/profile/profile.tsx

```diff
@@ -312,21 +379,49 @@ const Profile = () => {
             </Button>
           </Card.Body>
         </Card>
-        <Card>
+        <Card className="mb-3">
           <Card.Header>Dataset access</Card.Header>
           <Card.Body>
-            <div>
+            <p>
               {numDatasets ? (
                 <NavLink to="/work-package">
                   You have access to
                   {numDatasets === 1
                     ? " one dataset"
                     : ` ${numDatasets} datasets`}
-                  .
+                  .<br />
+                  Click here to set up your download tokens.
                 </NavLink>
               ) : (
-                <span>You do not yet have access to any datasets.</span>
+                <p>You do not yet have access to any datasets.</p>
               )}
+            </p>
+          </Card.Body>
+        </Card>
+        <Card>
+          <Card.Header>Pending access requests</Card.Header>
+          <Card.Body>
+            <div>
+              You have the following pending access requests:
+              <br />
+              <ul>
+                {userRequests?.map((x) => (
+                  <li key={x.id} className="mb-0">
+                    <Row>
+                      <Col md={6} lg={4} xxl={3}>
+                        For dataset{" "}
+                        <Link to={`/browse/${x.dataset_id}`}>
+                          {x.dataset_id}
+                        </Link>{" "}
+                      </Col>
+                      <Col md={3} xl={2}>
+                        from {x.access_starts.slice(0, 10)}
+                      </Col>
+                      <Col>to {x.access_ends.slice(0, 10)}</Col>
```

Comment:
I wouldn't put the "to" in a separate column, looks awkward.

Reply:
It's not aligned properly (since the font is not monospaced) otherwise